### PR TITLE
soc: arm: stm32l4: Add STM32L4Q5 support

### PR DESCRIPTION
**Pull request by zapol**
Add STM32L4Q5 to the list of supported SOCs.
Fix copy-paste error in STM32L4P5 SOC file.

### CODE REVIEW

File: soc/arm/st_stm32/stm32l4/Kconfig.defconfig.stm32l4q5xx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# ST Microelectronics STM32L4Q5xx MCU
+
+# Copyright (c) 2018 Pushpal Sidhu
```
Comment (FRASTM) [inline soc/arm/st_stm32/stm32l4/Kconfig.defconfig.stm32l4q5xx:3]: _Copyright (c) 2023 ..._

**Review by FRASTM — DISMISSED** [review]

File: soc/arm/st_stm32/stm32l4/Kconfig.defconfig.stm32l4q5xx (hunk 1 above)
Comment (zapol) [reply soc/arm/st_stm32/stm32l4/Kconfig.defconfig.stm32l4q5xx]: Fixed

**Review by gautierg-st — APPROVED** [review]

**Review by erwango — APPROVED** [review]

**Review by FRASTM — APPROVED** [review]

**Review by erwango — CHANGES_REQUESTED** [review]
Sorry if I missed this initially, but what about the .dtsi files ?

**Comment by erwango** [comment]
> Is something wrong with the one that's already in zephyr repo?

Sorry, these files are indeed actually already available, so you're actually fixing the SoC file misalignment. Perfect then. Approved. Enjoy your holidays!

**Review by erwango — APPROVED** [review]

**Pull request merged**